bot.js

```diff
@@ -18,7 +18,7 @@ var helpText = "Tell me to say something by\n" +
 var deniedText = "I'm sorry, I will not say that.";
 
 bot.on('message', message => {
-    if (message.content.startsWith('cowsay')) {
+    if (message.content.toLowerCase().startsWith('cowsay')) {
```

Comment:
I hate to be picky on such a small change but I think it would be better if we didn't toLower the entire string at this point. Remember this _is_ the prefix check so it happens for every message on every channel the bot is in. WIth that in mind, I think it would be faster to check `startsWith('cowsay') || startsWith('Cowsay')`, that way it will break early when possible.

Reply:
Sounds good will revert it back startsWith('cowsay') || startsWith('Cowsay')